src/amo/components/AddonDetail.js

```diff
@@ -161,6 +161,10 @@ export class AddonDetailBase extends React.Component {
     const authorList = addon.authors.map(
       (author) => `<a href="${author.url}">${author.name}</a>`);
 
+    const descriptionSanitized = addon.description && addon.description.length ?
+      sanitizeHTML(nl2br(addon.description), allowedDescriptionTags) :
```

Comment:
I think you should pull out `sanitizeHTML(nl2br(...), allowedDescriptionTags)` into a new variable on the next line to avoid repetition.